### PR TITLE
Add a helper tool to download Python.exe in build_windows.dockerfile

### PR DESCRIPTION
**Pull request by mrpau-julius**
### Summary

@benjaoming 
Add a helper tool to download python executable. 
The Dockerfile in Kolibri can call `make <target>` to download python.exe

### References

#4590
https://github.com/learningequality/kolibri-installer-windows/pull/92
### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Gherkin stories have been updated
- [ ] Unit tests have been updated

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.rst

### CODE REVIEW

File: docker/build_windows.dockerfile [hunk 1]
```diff
@@ -15,7 +15,9 @@ RUN git lfs install
 RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
 RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
       ttf-mscorefonts-installer \
-      wine-stable
+      wine-stable \
+      make \
+      aria2
```
Comment (benjaoming) [inline docker/build_windows.dockerfile]:
```suggestion
      wget
```

I think it's better to use `wget` for this. Using wget, you can also retry the download:

https://superuser.com/questions/493640/how-to-retry-connections-with-wget

**Review by benjaoming — DISMISSED** [review]

**Comment by rtibbles** [comment]
If this is intended for 0.12.0 please update the milestone and retarget to 0.12.x branch.

File: docker/build_windows.dockerfile (hunk 1 above)
Comment (mrpau-julius) [reply docker/build_windows.dockerfile]: Ok @benjaoming, thanks 👍. I will change that.

**Comment by benjaoming** [comment]
@rtibbles it is, let's try to get it in for the sake of a more stream-lined installer building process.

**Comment by indirectlylit** [comment]
remember to use the 'needs' labels to indicate what state a PR is in: it's not clear to me if this is ready for re-review

go ahead and dismiss the existing review if it's stale

File: docker/build_windows.dockerfile [hunk 2]
```diff
@@ -13,9 +13,12 @@ RUN apt-get update -y && \
 RUN git lfs install
 
 RUN echo ttf-mscorefonts-installer msttcorefonts/accepted-mscorefonts-eula select true | debconf-set-selections
-RUN DEBIAN_FRONTEND=noninteractive apt-get install -y \
+RUN RUN apt-get update -y && \
```
Comment (mrpau-julius) [inline docker/build_windows.dockerfile]: Hi @benjaoming, there is duplicate `RUN command`.

Comment (benjaoming) [reply docker/build_windows.dockerfile]: Sorry about this, just fixed now!

**Comment by indirectlylit** [comment]
removing milestone

**Comment by mrpau-julius** [comment]
@benjaoming I see from the log output the `env.list` contain `KOLIBRI_WINDOWS_INSTALLER_VERSION=v1.2.1` because of that, the built are failed. Can I change the version and add it in this PR? In order to check if the PR is running with no error on Buildkite. 
```
HEAD is now at 6341241 Merge pull request #90 from learningequality/develop
--
  | make: *** No targets specified and no makefile found.  Stop.
  | make: *** [docker-windows] Error 2

export KOLIBRI_VERSION=$(cat kolibri/VERSION) && \
--
  | docker run --env-file ./docker/env.list -v $PWD/dist:/kolibridist "learningequality/kolibri-windows"
  | Cloning into 'kolibri-installer-windows'...
  | Note: checking out 'v1.2.1'.
  |  
  | You are in 'detached HEAD' state. You can look around, make experimental
  | changes and commit them, and you can discard any commits you make in this
  | state without impacting any branches by performing another checkout.
  |  
  | If you want to create a new branch to retain commits you create, you may
  | do so (now or later) by using -b with the checkout command again. Example:
  |  
  | git checkout -b <new-branch-name>
  |  
  | HEAD is now at 6341241 Merge pull request #90 from learningequality/develop
  | make: *** No targets specified and no makefile found.  Stop.
  | make: *** [docker-windows] Error 2
  | 🚨 Error: The command exited with status 2


```

**Comment by indirectlylit** [comment]
@mrpau-julius I changed the target to a new feature branch `windows-build-test`. Feel free to set `KOLIBRI_WINDOWS_INSTALLER_VERSION` to whatever you need to for testing purposes.

**Comment by mrpau-julius** [comment]
Thanks, @indirectlylit. I test this in Windows 7 and 10, the installer is fully functional.

**Comment by benjaoming** [comment]
This looks good!

`KOLIBRI_WINDOWS_INSTALLER_VERSION` will be updated once https://github.com/learningequality/kolibri-installer-windows/pull/93 is merged.

**Comment by indirectlylit** [comment]
the `windows-build-test` was just created to test the build. @mrpau-julius since this got merged you'll need to create new a PR  from either `learningequality:windows-build-test` or `mrpau:hotfix/dovelop` to `learningequality:develop`

**Comment by indirectlylit** [comment]
ah, I see @benjaoming did that here https://github.com/learningequality/kolibri/pull/5330

**Pull request merged**